index.js

```diff
@@ -1,27 +1,117 @@
-import simpleGit from 'simple-git';
-import os from 'os';
-import prompts from 'prompts';
-
-const questions = [
-  {
-    type: 'text',
-    name: 'email',
-    message: 'Zadejte svůj email',
-  },
-  {
-    type: 'text',
-    name: 'name',
-    message: 'Zadejte své celé jméno',
-  },
-];
-
-console.info(`Operační systém: ${os.type()} ${os.arch()} ${os.release()}\n`);
-
-const response = await prompts(questions);
+import simpleGit from "simple-git";
+import os from "os";
+import prompts from "prompts";
+import { gte } from "semver";
+import kleur from "kleur";
+import { config } from "./config.js";
+//import config from "./config.json" assert {type: "json"}; // Import JSONu je zatím v Node.js experimentální a hlásí při startu varování, což je v tomto případě zbytečně matoucí.
```

Comment:
A nemusí fungovat ve starším node vůbec. :)

Reply:
> A nemusí fungovat ve starším node vůbec. :)

Říkal jsem si, jestli to nebude třeba ten důvod, proč vyžadovat Node.js ve verzi 18, ale i tam to vypisuje varování ;-)

Reply:
No hlavně nejde kontrolovat verzi Node.js kódem, který je napsaný jen pro nejnovější Node.js. :D